src/main/python/systemml/converters.py

```diff
@@ -19,18 +19,21 @@
 #
 #-------------------------------------------------------------
 
-__all__ = [ 'getNumCols', 'convertToMatrixBlock', 'convert_caffemodel', 'convert_lmdb_to_jpeg', 'convertToNumPyArr', 'convertToPandasDF', 'SUPPORTED_TYPES' , 'convertToLabeledDF', 'convertImageToNumPyArr']
+__all__ = [ 'getNumCols', 'convertToMatrixBlock', 'convert_caffemodel', 'convert_lmdb_to_jpeg', 'convertToNumPyArr', 'convertToPandasDF', 'SUPPORTED_TYPES' , 'convertToLabeledDF', 'convertImageToNumPyArr', 'getModelMean']
 
 import numpy as np
 import pandas as pd
 import os
 import math
+
 from pyspark.context import SparkContext
 from scipy.sparse import coo_matrix, spmatrix, csr_matrix
 from .classloader import *
 
 SUPPORTED_TYPES = (np.ndarray, pd.DataFrame, spmatrix)
 
+MODEL_MEAN = {'VGG-19':[103.939, 116.779, 123.68]}
```

Comment:
OK, that makes sense. Then, I would recommend calling it `getDatasetMean` and `DATASET_MEAN = {'ImageNet...SoAndSoYear':[103.939, 116.779, 123.68]}`